index.html

```diff
@@ -0,0 +1,216 @@
+          <a href="./index.html" class="org-logo"
+            ><img
+              src="./Images/logo/marvel-logo_footer-96x43.gif"
+              alt=""
+              class="org-logo-footer"
+          /></a>
+        </div>
+        <ul class="navbar-navlist">
+          <li class="navlist">
+            <a href="./about.html" class="navlist-links">About</a>
+          </li>
+          <li class="navlist">
+            <a href="#main-program" class="navlist-links">Program</a>
+          </li>
+          <li class="navlist"><a href="#" class="navlist-links">Join</a></li>
+          <li class="navlist"><a href="#" class="navlist-links">Sponsor</a></li>
+          <li class="navlist"><a href="#" class="navlist-links">News</a></li>
+          <li class="navlist-dif">
+            <a href="#" class="navlist-links-dif">Marvel</a>
+          </li>
+          <li class="navlist-logo">
+            <a href="./index.html" class="org-logo"
+              ><img
+                src="./Images/logo/marvel-logo_footer-96x43.gif"
+                alt=""
+                class="org-logo-footer"
+            /></a>
+          </li>
+        </ul>
+      </div>
+    </nav>
+
+    <!-- head secton with some info -->
+    <section id="header" class="intro-greeting-info general-container">
+      <h2 class="greeting">“I'd like to welcome everyone”</h2>
+      <h1 class="head-title">Marvel Divide 2015</h1>
+      <p class="head-title-info">
+        The precursor to Marvel Comics was founded in 1939 by pulp magazine
+        publisher Martin Goodman. In order to capitalize on the growing
+        popularity of comic books—especially those starring superheroes—Goodman
+        created Timely Comics. Timely’s first comic book was Marvel Comics no. 1
+        (cover dated October 1939), which featured several superhero characters,
+        most notably the Human Torch and the Sub-Mariner. Timely Comics
+        introduced many superhero characters during comics’ “Golden Age” in the
+        1940s, most importantly Captain America, who first appeared in Captain
+        America Comics no. 1 (March 1941). Timely characters were often
+        portrayed as fighting against the Nazis and the Japanese even before the
+        United States entered World War II. As the 1940s came to a close,
+        superheroes fell out of vogue with comic book readers, and Timely
+        canceled the last of its books in this genre in 1950. In 1951 Goodman
+        formed his own distribution company, and Timely Comics became Atlas
+        Magazines. Though there was a brief experiment in bringing back
+        superheroes such as Captain America in 1953, Atlas’s output was mostly
+        in other genres such as humour, westerns, horror, war, and science
+        fiction.
+      </p>
+      <div class="time-line">2015.10.15(THU) ~ 16(FRI)</div>
+      <div class="location">
+        @ Royal Victoria Dock 1 Western Gateway London E16 1XL London, UK
+      </div>
+    </section>
```

Comment:
- [ ] [REQUIRED] Good job so far! You forgot to make the elements shown at the `top` of the [Behance design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015) on the main page. Please create it!

![image](https://user-images.githubusercontent.com/63915024/192008485-f8465622-2fe1-452d-ab37-793dc4fb4d60.png)

Reply:
thank you for the correction 👍